### ISSUE TITLE
Improvements to night mode

### ISSUE THREAD

**Issue by Akianonymus** [kiwibrowser/src#91]
- Ability to increase brightness of texts in a webpage.

Although it makes sense to to have a decreased focus on text as the feature is "night mode", but i am sure many people including myself use it always. So kinda difficult to read.

Currently tackling this problem by using Darkreader extension on github, but maybe this can be implemented in browser natively.

Here are two screenshots, before and after

Before applying Darkreader:
![Screenshot_20200515-101635](https://user-images.githubusercontent.com/32216346/82012433-94440280-9695-11ea-813a-f5f1c704d3cb.png)

After applying Darkreader:
![Screenshot_20200515-101838](https://user-images.githubusercontent.com/32216346/82012462-acb41d00-9695-11ea-9b09-27a373c76d26.png)

This can also be achived by increasing the contrast to a really high value, but that makes the webpages even more black.

So, basically i want to configure contrast and brightness individually ( Darkreader have it ).

.

- Ability to blacklist URL in night mode

Do not darken webpages which are blacklisted. Helpful when a website already have a dark mode.

**Comment by d3ward** [comment]
This wasn't a priority and neither now .
But you have to check out latest build and see if the new night mode is ok for you 
Consider open a new issue on [src.next](https://github.com/kiwibrowser/src.next) if want to suggest improvements
